quality/postman/emodb_tests.postman_collection.json

```diff
@@ -3827,11 +3835,21 @@
 																	"    pm.response.to.have.status(200);",
 																	"});",
 																	"",
-																	"const stringToReplace = '{\"availability\":{\"facade\":false,\"placement\":\"ugc_global:ugc\"},\"name\":\"postman_test_table\",\"options\":{\"facades\":[],\"placement\":\"ugc_global:ugc\"},\"template\":{}}';",
-																	"const expectedResponceBody = stringToReplace.replace(/postman_test_table/, pm.environment.get(\"table12\"));",
+																	"const postman_test_table = pm.environment.get(\"table12\");",
+																	"const expectedResponceBody = {",
+																	"    \"availability\":{",
+																	"        \"facade\":false,",
+																	"        \"placement\":\"ugc_global:ugc\"",
```

Comment:
does it make sense to extract placement to postman env variables?

Reply:
https://bits.bazaarvoice.com/jira/browse/EMO-7633 has been create for this